tasks/build_tags.py

```diff
@@ -250,9 +250,9 @@ def compute_build_tags_for_flavor(
 
 
 @task
-def print_default_build_tags(_, build="agent", flavor=AgentFlavor.base.name):
+def print_default_build_tags(_, build="agent", flavor=AgentFlavor.base.name, platform=sys.platform):
```

Comment:
Note that you can use `platform = platform or sys.platform` in this example which makes the code clearer IMO

Reply:
I agree, but I did it that way because it's the default value of the `get_default_build_tag` function already
```
def get_default_build_tags(build="agent", flavor=AgentFlavor.base, platform=sys.platform):
```

I'll update both functions to use `None` as default value value, it will make it cleaner 👍

Reply:
I don't find this clearer, but to each their own

Reply:
Yes I figured, thanks for changing them both!